### PR TITLE
test(menu-select): common test suite

### PR DESCRIPTION
**Pull request by aymeric-giraudet**
**Summary**

There weren't many logic tests to remove, probably because they're normally tested in the connector (so `connectMenu`)
Also obviously had to skip testing for React as it doesn't have `MenuSelect`

### CODE REVIEW

File: tests/common/widgets/menu-select/index.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import type { MenuSelectWidget } from 'instantsearch.js/es/widgets/menu-select/menu-select';
+
+type WidgetParams = Parameters<MenuSelectWidget>[0];
+export type MenuSelectWidgetSetup = TestSetup<
+  {
+    widgetParams: Omit<WidgetParams, 'container'>;
+  },
+  {
+    flavor: string;
+  }
```
Comment (dhayab) [inline tests/common/widgets/menu-select/index.ts:15]: We don't need this here, do we?

File: packages/instantsearch.js/src/widgets/menu-select/__tests__/menu-select.test.tsx
Comment (dhayab) [inline packages/instantsearch.js/src/widgets/menu-select/__tests__/menu-select.test.tsx:1]: Can you add a test for `cssClasses`?

Comment (aymeric-giraudet) [reply packages/instantsearch.js/src/widgets/menu-select/__tests__/menu-select.test.tsx:1]: Can't find other CTS that tests this, when I change it it doesn't seem to be taken into account for vue

Comment (dhayab) [reply packages/instantsearch.js/src/widgets/menu-select/__tests__/menu-select.test.tsx:1]: Not in CTS, in the JS widget tests. You can look at hits or current refinements for example. We should have done it for all CTS migrated widgets but it seems we forgot a few on the way when I searched for `adds custom CSS classes` 😓 .

Comment (aymeric-giraudet) [reply packages/instantsearch.js/src/widgets/menu-select/__tests__/menu-select.test.tsx:1]: Ah yes, done 👍

**Review by Haroenv — APPROVED** [review]

**Review by dhayab — APPROVED** [review]

**Pull request merged**